Ports/.port_include.sh

```diff
@@ -95,6 +96,36 @@ ensure_build() {
     fi
 }
 
+install_main_icon() {
+    if [ -n "$icon_file" ] && [ -n "$launcher_command" ]; then
+	install_icon "$icon_file" "$launcher_command"
```

Comment:
This mixes tabs and spaces. Please don't.

Reply:
Good catch, I'll work with editor to prevent this in the future! Fixed.